interface/drugs/add_edit_drug.php

```diff
@@ -372,263 +368,235 @@ function sel_related() {
+                    "]' value='" . attr(0 + $pwrow['pw_max_level']) . "' size='5' " .
+                    "title='" . xla('Warehouse maximum, 0 if not applicable') . "' data-toggle='tooltip' data-placement='top' />";
+                    echo "</td>\n";
+                }
+                ?>
+            </tr>
+        </table>
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Pill Size'); ?>:</label>
+        <?php
+            generate_form_field(array('data_type' => 1,'field_id' => 'form','list_id' => 'drug_form','empty_title' => 'SKIP'), $row['form']);
+        ?>
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Form'); ?>:</label>
+        <input class="form-control" size="5" name="form_size" maxlength="7" value='<?php echo attr($row['size']) ?>' />
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Units'); ?>:</label>
+        <?php
+            generate_form_field(array('data_type' => 1,'field_id' => 'unit','list_id' => 'drug_units','empty_title' => 'SKIP'), $row['unit']);
+        ?>
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Route'); ?>:</label>
+        <?php
+            generate_form_field(array('data_type' => 1,'field_id' => 'route','list_id' => 'drug_route','empty_title' => 'SKIP'), $row['route']);
+        ?>
+    </div>
+
+    <div class="form-group mt-3 ippfonly">
+        <label class="font-weight-bold"><?php echo xlt('CYP Factor'); ?>:</label>
+        <input class="form-control" size="10" name="form_cyp_factor" maxlength="20" value='<?php echo attr($row['cyp_factor']) ?>' />
+    </div>
+
+    <div class="form-group mt-3 drugsonly">
+        <label class="font-weight-bold"><?php echo xlt('Relate To'); ?>:</label>
+        <input class="form-control w-100" type="text" size="50" name="form_related_code" value='<?php echo attr($row['related_code']) ?>'
+             onclick='sel_related()' title='<?php echo xla('Click to select related code'); ?>' data-toggle="tooltip" data-placement="top" readonly />
+    </div>
+
+    <div class="form-group mt-3">
+        <label class="font-weight-bold">
+            <?php echo $GLOBALS['sell_non_drug_products'] == 2 ? xlt('Fees') : xlt('Templates'); ?>:
+        </label>
+        <table class='table table-borderless'>
+            <tr>
+                <td class='drugsonly font-weight-bold'><?php echo xlt('Name'); ?></td>
+                <td class='drugsonly font-weight-bold'><?php echo xlt('Schedule'); ?></td>
+                <td class='drugsonly font-weight-bold'><?php echo xlt('Interval'); ?></td>
+                <td class='drugsonly font-weight-bold'><?php echo xlt('Qty'); ?></td>
+                <td class='drugsonly font-weight-bold'><?php echo xlt('Refills'); ?></td>
```

Comment:
What does this `drugsonly` class do?

Reply:
I think it using PHP to check display status, if `$GLOBALS['sell_non_drug_products'] == 2`  is true, the `.drugsonly` will be enable.

https://github.com/openemr/openemr/blob/master/interface/drugs/add_edit_drug.php#L88-L92